src/wormhole_transit_relay/test/test_rlimits.py

```diff
@@ -1,8 +1,5 @@
 from __future__ import print_function, unicode_literals
```

Comment:
Can't these also be dropped if we are dropping python2?

Reply:
Ahh, yeah good catch. Thanks!

Reply:
okay, dropped all `__future__` imports